daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/VersionTimelineSpec.scala

```diff
@@ -73,7 +73,8 @@ class VersionTimelineSpec
       inside(inAscendingOrder.last) {
         case Both(
             Both(ValueVersion("dev"), TransactionVersion("dev")),
```

Comment:
We should probably do the same for value and transactions versions as well at some point.

Reply:
:+1:

Reply:
I include it in the PR.